.ebextensions/001_serversettings.config

```diff
@@ -18,7 +18,7 @@ commands:
   08-command:
     command: "/usr/local/bin/npm install -g coffee-script@1.12.2 || echo coffeescript already installed"
   09-command:
-    command: "rm -rf /usr/local/go && curl -s https://storage.googleapis.com/golang/go1.7.1.linux-amd64.tar.gz | tar -v -C /usr/local -xz"
+    command: "rm -rf /usr/local/go && curl -s https://storage.googleapis.com/golang/go1.8.linux-amd64.tar.gz | tar -v -C /usr/local -xz"
```

Comment:
infra repo + Dockerfile needs to be updated as well. Prob new docker image for CircleCI too /cc @szkl